inc/osvr/PluginKit/EyeTrackerInterfaceC.h

```diff
@@ -0,0 +1,92 @@
+/** @file
+    @brief Header
+
+    @date 2015
+
+    @author
+    Sensics, Inc.
+    <http://sensics.com/osvr>
+
+*/
+
+// Copyright 2015 Sensics, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// 	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef INCLUDED_EyeTrackerInterfaceC_h_GUID_F6C50F38_5C4F_41AD_B703_DE8A073F77B3
+#define INCLUDED_EyeTrackerInterfaceC_h_GUID_F6C50F38_5C4F_41AD_B703_DE8A073F77B3
+
+/* Internal Includes */
+#include <osvr/PluginKit/DeviceInterfaceC.h>
+#include <osvr/Util/ChannelCountC.h>
+#include <osvr/Util/EyeTrackerReportTypesC.h>
+
+/* Library/third-party includes */
+/* none */
+
+/* Standard includes */
+/* none */
+
+/** @defgroup PluginKitCEyeTracker EyeTracker interface underlying C API - not
```

Comment:
Eyetracker messages are fine to use with the C API - the main reason this isn't true for imaging is because the wrapper extracts data from the OpenCV data structures before passing it into C.